cmd/config.go

```diff
@@ -63,6 +95,17 @@ func (d *DBConfig) URL() (string, error) {
 	return d.DBConnect, nil
 }
 
+// GetMaxOpenConns defaults MaxOpenConns to the value of MaxDBConns if
+// MaxDBConns was specified in the config json.
+// TODO(#5249): This method can be removed once MaxDBConns has been
+// removed from test/config and all prod and staging configs.
+func (d *DBConfig) GetMaxOpenConns() int {
+	if d.MaxDBConns != 0 {
```

Comment:
I think what we usually do for these sorts of things is "default to using the new setting. if the new value isn't set, then fall back to the old setting". So this would actually be
```
if d.MaxOpenConns == 0 {
  d.MaxOpenConns = d.MaxDBConns
}
```

Reply:
Ah, thanks for pointing this out. I'll ship a small revision commit :)